README.md

```diff
@@ -86,33 +68,41 @@ details. In addition Kubernetes itself doesn't yet support sub-resources
 for custom resource definitions. As a result the provider has to extract
 the SPICE proxy URL from the configuration of the `spice-proxy` service.
 
-== Notes
+## Notes
 
-=== How to get the default token from Kubernetes
+### How to get the default token from Kubernetes
 
 List the set of secrets:
 
+  ```
```

Comment:
For code blocks you can just change the indentation from 2 spaces to 4 spaces. If you are going to use the three backticks instead, then I'd suggest that you remove the indentation completely.

Reply:
I checked this and it seems not to work so I added the line that you commented.

Reply:
Here is [code quoting](https://help.github.com/articles/basic-writing-and-formatting-syntax/#quoting-code) guideline.